sdk/search/Azure.Search.Documents/src/SearchClient.cs

```diff
@@ -1899,5 +1899,26 @@ public class SearchClient
             }
         }
         #endregion Index Documents Conveniences
+
+        /// <summary>
+        /// Creates a new <see cref="SearchIndexingBufferedSender{T}"/> that
+        /// can be used to index search documents with intelligent batching,
+        /// automatic flushing, and retries for failed indexing actions.
+        /// </summary>
+        /// <typeparam name="T">
+        /// The .NET type that maps to the index schema.  Instances of this
+        /// type can be retrieved as documents from the index. You can use
+        /// <see cref="SearchDocument"/> for dynamic documents.
+        /// </typeparam>
+        /// <param name="options">
+        /// The <see cref="SearchIndexingBufferedSenderOptions{T}"/> to
+        /// customize the sender's behavior.
+        /// </param>
+        /// <returns>
+        /// A new <see cref="SearchIndexingBufferedSender{T}"/>.
+        /// </returns>
+        public virtual SearchIndexingBufferedSender<T> CreateIndexingBufferedSender<T>(
```

Comment:
So this is back in? I thought it was removed since the archboard requested it. If so, we need to figure out the problem. Ultimately, the tests were still tied to the `SearchIndexingBufferedSender` class either way. And the pipeline was reused either way, IIRC, so I'm not sure why the problems. When you were testing, did you test locally against the recordings like the PR validation would've?

Reply:
Here's the situation - there are multiple items identified as part of supporting batching in a better way. One of them is to remove this helper method. However, that change has a cascading effect and causes a bunch of tests to fail, and we don't understand why.
So, as a mitigation I am removing that part of the change and keeping the rest. A subsequent PR would focus on removing the helper method and dealing with the test failures.